operator/src/main/java/oracle/kubernetes/operator/helpers/CallBuilder.java

```diff
@@ -65,8 +60,6 @@
   /** HTTP status code for "Not Found". */
   public static final int NOT_FOUND = 404;
 
-  private static final LoggingFacade LOGGER = LoggingFactory.getLogger("Operator", "Operator");
```

Comment:
The class simply creates calls to Kubernetes. The previous change had added a logger because it had improperly created application logic inside the class. It never had logging before and shouldn't need it now.